vscode-antimony/src/server/stibium/stibium/antimony.lark

```diff
@@ -12,6 +12,33 @@ species : [NUMBER] ["$"] NAME
 ARROW : "->"
     | "=>"
 
+event : [reaction_name] "at" [event_delay] [event_condition_list] [event_trigger_list] ":" event_assignment_list
+
+event_delay : sum "after"
+
+event_condition_list : [parenthesis_list] event_condition [parenthesis_list] (logical [parenthesis_list] event_condition [parenthesis_list])*
+event_condition : sum compare_sign sum

```

Comment:
This is actually incorrect:  the event trigger can be any function, which I think you have defined as 'sum'.  It will usually return a Boolean, but it doesn't need to!  It can be numeric, and merely be *interpreted* as a boolean in SBML.  So you could have have an event trigger "at S1" that would fire whenever S1 changed from 0 to anything else.  You could have it be a function:  "at check(S1, k1)" which would perform some test on S1 and k1 and return true or false.  You could even say "at true" which would never fire, as it would never transition from false to true...unless it was set t0=false, in which case it would trigger at t0.  Anyway, the compare_sign shouldn't be used here; just 'sum'.  Assuming 'sum' includes compare_signs (plus '==').  And if sum doesn't already include them, it should!